lib/java/com/google/android/material/textfield/IndicatorViewController.java

```diff
@@ -361,20 +364,20 @@ void adjustIndicatorPadding() {
       boolean isFontScaleLarge = MaterialResources.isFontScaleAtLeast1_3(context);
       ViewCompat.setPaddingRelative(
           indicatorArea,
+          getIndicatorHorizontalPadding(
+            isFontScaleLarge,
+            R.dimen.material_helper_text_font_1_3_padding_horizontal,
+            ViewCompat.getPaddingStart(editText)),
           getIndicatorPadding(
```

Comment:
I feel we probably need some refactoring here. 

To make the code more clear, I think we should create separate methods `getHelperTextVerticalPadding` and `getHelperTextHorizontalPadding`, and call `getIndicatorPadding` from them.

Reply:
I didn't catch your idea. We have only horizontal padding option for now. What is the purpose of `getHelperTextVerticalPadding` then? `getIndicatorHorizontalPadding` encapsulates padding resolution logic. Can  you reformulate your suggestion?